inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -55,6 +56,21 @@ std::string translate_type_name(const std::string& name) {
     return name;
 }
 
+// Some of the operators were added to wrong opsets. This is a
+// mapping that allows such operators to be serialized with proper
+// opsets. If new operators are discovered that have the same problem
+// the opset mapping needs to be updated here.
+const std::unordered_map<std::string, std::string>
```

Comment:
Maybe it can be a static variable inside the `get_special_opset_for_op` function

Reply:
I think it is good to keep the same convention with translate_type_name_translator - currently is global at the top of the file (previously translate_type_name_translator was static inside function but was moved to the top). So, let's establish one convention and stick to it in the future.

Reply:
Agree, but we already have two approaches ([For example](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/readers/ir_reader/ie_ir_parser.cpp#L625)) 😄

Reply:
So, let's establish a convention now so this is clear. I could move the map for type name translator and opset mapping into their appropriate functions (so we have one approach for these two similar functionalities), or leave it as it is.

Reply:
I'm resolving this conversation as there seems to be no conclusion so far on the desired approach.